Helmsman/dsf/esignet-standalone/external-dsf.yaml

```
@@ -76,25 +79,124 @@ apps:
+      # ---------------------------------------------------------------------------
+      # esignet-mosipid1 namespace (plugin 2 - mosip-identity)
+      # ---------------------------------------------------------------------------
+      databases.mosip_esignet_mosipid1.enabled: "true"
+      databases.mosip_esignet_mosipid1.scriptsDir: "mosip_esignet"
+      databases.mosip_esignet_mosipid1.dbName: "mosip_esignet_mosipid1"
+      databases.mosip_esignet_mosipid1.dbUser: "esignetuser_mosipid1"
+      databases.mosip_esignet_mosipid1.branch: "develop"
+      databases.mosip_esignet_mosipid1.host: "postgres.${domain_name}"
+      databases.mosip_esignet_mosipid1.port: ${esignet_db_port}
+      databases.mosip_esignet_mosipid1.dml: 1
+      databases.mosip_esignet_mosipid1.repoUrl: "https://github.com/mosip/esignet.git"
```

YAMLlint (1.37.1)

[error] duplication of key "databases.mosip_esignet_mosipid1.repoUrl" in mapping

(key-duplicates)